2025-05-13 17:59:39,483 ERROR: Erro ao cadastrar m치quina: (pymysql.err.IntegrityError) (1062, "Duplicate entry '8146' for key 'registros.ix_registros_ramal'")
[SQL: INSERT INTO registros (base, nome, departamento, endereco_ip, mac_adress, hostname, memoria_ram, ssd, ramal, anydesk, data_cadastro, ultima_atualizacao) VALUES (%(base)s, %(nome)s, %(departamento)s, %(endereco_ip)s, %(mac_adress)s, %(hostname)s, %(memoria_ram)s, %(ssd)s, %(ramal)s, %(anydesk)s, %(data_cadastro)s, %(ultima_atualizacao)s)]
[parameters: {'base': 'SP', 'nome': 'maykon.oliveira', 'departamento': 'Fiscal', 'endereco_ip': '10.0.225.206', 'mac_adress': 'D0:94:66:B0:1B:27', 'hostname': '40Z54W2', 'memoria_ram': 8, 'ssd': 256, 'ramal': 8146, 'anydesk': 1541656676, 'data_cadastro': datetime.datetime(2025, 5, 13, 17, 59, 39, 477712), 'ultima_atualizacao': datetime.datetime(2025, 5, 13, 17, 59, 39, 477721)}]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in /var/www/Portal-TI-Manager/app.py:305]
2025-05-13 17:59:48,278 ERROR: Erro ao cadastrar m치quina: (pymysql.err.IntegrityError) (1062, "Duplicate entry '8146' for key 'registros.ix_registros_ramal'")
[SQL: INSERT INTO registros (base, nome, departamento, endereco_ip, mac_adress, hostname, memoria_ram, ssd, ramal, anydesk, data_cadastro, ultima_atualizacao) VALUES (%(base)s, %(nome)s, %(departamento)s, %(endereco_ip)s, %(mac_adress)s, %(hostname)s, %(memoria_ram)s, %(ssd)s, %(ramal)s, %(anydesk)s, %(data_cadastro)s, %(ultima_atualizacao)s)]
[parameters: {'base': 'SP', 'nome': 'maykon.oliveira', 'departamento': 'Fiscal', 'endereco_ip': '10.0.225.206', 'mac_adress': 'D0:94:66:B0:1B:27', 'hostname': '40Z54W2', 'memoria_ram': 8, 'ssd': 256, 'ramal': 8146, 'anydesk': 1541656676, 'data_cadastro': datetime.datetime(2025, 5, 13, 17, 59, 48, 272674), 'ultima_atualizacao': datetime.datetime(2025, 5, 13, 17, 59, 48, 272674)}]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in /var/www/Portal-TI-Manager/app.py:305]
2025-05-13 18:01:29,942 ERROR: Erro ao cadastrar m치quina: (pymysql.err.IntegrityError) (1062, "Duplicate entry '8184' for key 'registros.ix_registros_ramal'")
[SQL: INSERT INTO registros (base, nome, departamento, endereco_ip, mac_adress, hostname, memoria_ram, ssd, ramal, anydesk, data_cadastro, ultima_atualizacao) VALUES (%(base)s, %(nome)s, %(departamento)s, %(endereco_ip)s, %(mac_adress)s, %(hostname)s, %(memoria_ram)s, %(ssd)s, %(ramal)s, %(anydesk)s, %(data_cadastro)s, %(ultima_atualizacao)s)]
[parameters: {'base': 'SP', 'nome': 'jessica.marcurci', 'departamento': 'ComGer', 'endereco_ip': '10.0.225.20', 'mac_adress': '74:86:7A:F4:ED:1B', 'hostname': '1XZQ9X1', 'memoria_ram': 8, 'ssd': 222, 'ramal': 8184, 'anydesk': 1868199339, 'data_cadastro': datetime.datetime(2025, 5, 13, 18, 1, 29, 936572), 'ultima_atualizacao': datetime.datetime(2025, 5, 13, 18, 1, 29, 936581)}]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in /var/www/Portal-TI-Manager/app.py:305]
2025-05-13 18:01:36,800 ERROR: Erro ao cadastrar m치quina: (pymysql.err.IntegrityError) (1062, "Duplicate entry '8184' for key 'registros.ix_registros_ramal'")
[SQL: INSERT INTO registros (base, nome, departamento, endereco_ip, mac_adress, hostname, memoria_ram, ssd, ramal, anydesk, data_cadastro, ultima_atualizacao) VALUES (%(base)s, %(nome)s, %(departamento)s, %(endereco_ip)s, %(mac_adress)s, %(hostname)s, %(memoria_ram)s, %(ssd)s, %(ramal)s, %(anydesk)s, %(data_cadastro)s, %(ultima_atualizacao)s)]
[parameters: {'base': 'SP', 'nome': 'jessica.marcurci', 'departamento': 'ComGer', 'endereco_ip': '10.0.225.20', 'mac_adress': '74:86:7A:F4:ED:1B', 'hostname': '1XZQ9X1', 'memoria_ram': 8, 'ssd': 222, 'ramal': 8184, 'anydesk': 1868199339, 'data_cadastro': datetime.datetime(2025, 5, 13, 18, 1, 36, 793702), 'ultima_atualizacao': datetime.datetime(2025, 5, 13, 18, 1, 36, 793711)}]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in /var/www/Portal-TI-Manager/app.py:305]
2025-05-13 18:05:16,143 ERROR: Erro ao cadastrar m치quina: (pymysql.err.IntegrityError) (1062, "Duplicate entry '8184' for key 'registros.ix_registros_ramal'")
[SQL: INSERT INTO registros (base, nome, departamento, endereco_ip, mac_adress, hostname, memoria_ram, ssd, ramal, anydesk, data_cadastro, ultima_atualizacao) VALUES (%(base)s, %(nome)s, %(departamento)s, %(endereco_ip)s, %(mac_adress)s, %(hostname)s, %(memoria_ram)s, %(ssd)s, %(ramal)s, %(anydesk)s, %(data_cadastro)s, %(ultima_atualizacao)s)]
[parameters: {'base': 'SP', 'nome': 'elias.oliveira', 'departamento': 'ComGer', 'endereco_ip': '10.0.225.32', 'mac_adress': 'D0:94:66:A8:8E:2D', 'hostname': 'JN2KSL2', 'memoria_ram': 8, 'ssd': 237, 'ramal': 8184, 'anydesk': 1749075813, 'data_cadastro': datetime.datetime(2025, 5, 13, 18, 5, 16, 136578), 'ultima_atualizacao': datetime.datetime(2025, 5, 13, 18, 5, 16, 136587)}]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in /var/www/Portal-TI-Manager/app.py:305]
2025-05-13 18:05:17,745 ERROR: Erro ao cadastrar m치quina: (pymysql.err.IntegrityError) (1062, "Duplicate entry '8184' for key 'registros.ix_registros_ramal'")
[SQL: INSERT INTO registros (base, nome, departamento, endereco_ip, mac_adress, hostname, memoria_ram, ssd, ramal, anydesk, data_cadastro, ultima_atualizacao) VALUES (%(base)s, %(nome)s, %(departamento)s, %(endereco_ip)s, %(mac_adress)s, %(hostname)s, %(memoria_ram)s, %(ssd)s, %(ramal)s, %(anydesk)s, %(data_cadastro)s, %(ultima_atualizacao)s)]
[parameters: {'base': 'SP', 'nome': 'elias.oliveira', 'departamento': 'ComGer', 'endereco_ip': '10.0.225.32', 'mac_adress': 'D0:94:66:A8:8E:2D', 'hostname': 'JN2KSL2', 'memoria_ram': 8, 'ssd': 237, 'ramal': 8184, 'anydesk': 1749075813, 'data_cadastro': datetime.datetime(2025, 5, 13, 18, 5, 17, 738238), 'ultima_atualizacao': datetime.datetime(2025, 5, 13, 18, 5, 17, 738247)}]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in /var/www/Portal-TI-Manager/app.py:305]
2025-05-13 18:05:18,438 ERROR: Erro ao cadastrar m치quina: (pymysql.err.IntegrityError) (1062, "Duplicate entry '8184' for key 'registros.ix_registros_ramal'")
[SQL: INSERT INTO registros (base, nome, departamento, endereco_ip, mac_adress, hostname, memoria_ram, ssd, ramal, anydesk, data_cadastro, ultima_atualizacao) VALUES (%(base)s, %(nome)s, %(departamento)s, %(endereco_ip)s, %(mac_adress)s, %(hostname)s, %(memoria_ram)s, %(ssd)s, %(ramal)s, %(anydesk)s, %(data_cadastro)s, %(ultima_atualizacao)s)]
[parameters: {'base': 'SP', 'nome': 'elias.oliveira', 'departamento': 'ComGer', 'endereco_ip': '10.0.225.32', 'mac_adress': 'D0:94:66:A8:8E:2D', 'hostname': 'JN2KSL2', 'memoria_ram': 8, 'ssd': 237, 'ramal': 8184, 'anydesk': 1749075813, 'data_cadastro': datetime.datetime(2025, 5, 13, 18, 5, 18, 432630), 'ultima_atualizacao': datetime.datetime(2025, 5, 13, 18, 5, 18, 432653)}]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in /var/www/Portal-TI-Manager/app.py:305]
2025-05-13 18:05:20,943 ERROR: Erro ao cadastrar m치quina: (pymysql.err.IntegrityError) (1062, "Duplicate entry '8184' for key 'registros.ix_registros_ramal'")
[SQL: INSERT INTO registros (base, nome, departamento, endereco_ip, mac_adress, hostname, memoria_ram, ssd, ramal, anydesk, data_cadastro, ultima_atualizacao) VALUES (%(base)s, %(nome)s, %(departamento)s, %(endereco_ip)s, %(mac_adress)s, %(hostname)s, %(memoria_ram)s, %(ssd)s, %(ramal)s, %(anydesk)s, %(data_cadastro)s, %(ultima_atualizacao)s)]
[parameters: {'base': 'SP', 'nome': 'elias.oliveira', 'departamento': 'ComGer', 'endereco_ip': '10.0.225.32', 'mac_adress': 'D0:94:66:A8:8E:2D', 'hostname': 'JN2KSL2', 'memoria_ram': 8, 'ssd': 237, 'ramal': 8184, 'anydesk': 1749075813, 'data_cadastro': datetime.datetime(2025, 5, 13, 18, 5, 20, 937874), 'ultima_atualizacao': datetime.datetime(2025, 5, 13, 18, 5, 20, 937883)}]
(Background on this error at: https://sqlalche.me/e/20/gkpj) [in /var/www/Portal-TI-Manager/app.py:305]
2025-05-13 18:10:20,796 INFO: Nova m치quina cadastrada: juliana.gomes (10.0.225.24) [in /var/www/Portal-TI-Manager/app.py:301]
2025-05-13 18:25:48,991 INFO: Exporta칞칚o de CSV gerada: maquinas_Portal_20250513_182548.csv [in /var/www/Portal-TI-Manager/app.py:508]
2025-05-13 16:52:51,228 INFO: Portal TI Manager - Iniciando aplicacao [in D:\ti-manager\app.py:72]
2025-05-13 16:52:51,343 INFO: Banco de dados inicializado [in D:\ti-manager\app.py:941]
2025-05-13 16:52:52,410 INFO: Portal TI Manager - Iniciando aplicacao [in D:\ti-manager\app.py:72]
2025-05-13 16:52:52,505 INFO: Banco de dados inicializado [in D:\ti-manager\app.py:941]
2025-05-13 17:05:39,895 INFO: Portal TI Manager - Iniciando aplicacao [in D:\portal-ti-manager\app.py:72]
2025-05-13 17:05:39,985 INFO: Banco de dados inicializado [in D:\portal-ti-manager\app.py:941]
2025-05-13 17:05:41,030 INFO: Portal TI Manager - Iniciando aplicacao [in D:\portal-ti-manager\app.py:72]
2025-05-13 17:05:41,104 INFO: Banco de dados inicializado [in D:\portal-ti-manager\app.py:941]
2025-05-13 17:15:45,922 INFO: Portal TI Manager - Iniciando aplicacao [in D:\portal-ti-manager\app.py:72]
2025-05-13 17:15:46,075 INFO: Banco de dados inicializado [in D:\portal-ti-manager\app.py:942]
2025-05-13 17:16:18,237 INFO: Portal TI Manager - Iniciando aplicacao [in D:\portal-ti-manager\app.py:72]
2025-05-13 17:16:18,330 INFO: Banco de dados inicializado [in D:\portal-ti-manager\app.py:943]
2025-05-13 17:22:33,727 INFO: Portal TI Manager - Iniciando aplicacao [in D:\portal-ti-manager\app.py:72]
2025-05-13 17:22:33,806 INFO: Banco de dados inicializado [in D:\portal-ti-manager\app.py:943]
2025-05-13 17:22:35,086 INFO: Portal TI Manager - Iniciando aplicacao [in D:\portal-ti-manager\app.py:72]
2025-05-13 17:22:35,269 INFO: Banco de dados inicializado [in D:\portal-ti-manager\app.py:943]
2025-05-13 17:23:29,539 INFO: Nova m瓊uina cadastrada: alexandro.gomes (10.0.225.23) [in D:\portal-ti-manager\app.py:302]
2025-05-13 17:24:48,094 INFO: Nova m瓊uina cadastrada: maykon.oliveira (10.0.225.206) [in D:\portal-ti-manager\app.py:302]
2025-05-13 17:25:46,662 INFO: M瓊uina exclu顔a: ID 40 - yasmin.silva [in D:\portal-ti-manager\app.py:444]
2025-05-13 17:25:49,069 INFO: M瓊uina exclu顔a: ID 47 - josefa.santana [in D:\portal-ti-manager\app.py:444]
